website/docs/cli/configuration/terminal.mdx

```diff
@@ -0,0 +1,146 @@
+
+Configure general terminal behavior:
+
+```yaml
+settings:
+  terminal:
+    max_width: 120          # Maximum width for terminal output
+    use_pager: true         # Whether to use a pager (like 'less') for long output
+    timestamps: false       # Show timestamps in output
+    colors: true           # Enable colored output
+    unicode: true          # Use Unicode characters in output
+```
+
+## Syntax Highlighting
+
+You can customize the syntax highlighting behavior for terminal output using the following settings:
+
+```yaml
+settings:
+  terminal:
+    syntax_highlighting:
+      enabled: true         # Enable/disable syntax highlighting
+      lexer: yaml           # Lexer to use (yaml or json)
+      formatter: terminal   # Output formatter
+      style: dracula        # Color scheme to use
+      use_pager: false      # Whether to use a pager for highlighted output
+      options:
+        line_numbers: false # Show line numbers
+        wrap: false         # Wrap long lines
+```
+
+### Configuration Options
+
+<dl>
+  <dt>`enabled`</dt>
+  <dd>Enable or disable syntax highlighting (default: `true`)</dd>
+  
+  <dt>`lexer`</dt>
+  <dd>
+    Specify the lexer to use for syntax highlighting:
+    <dl>
+      <dt>`yaml`</dt>
+      <dd>For YAML output</dd>
+      <dt>`json`</dt>
+      <dd>For JSON output</dd>
+    </dl>
+  </dd>
+  
+  <dt>`formatter`</dt>
+  <dd>Output formatter (default: `terminal`)</dd>
+  
+  <dt>`style`</dt>
+  <dd>
+    Color scheme for syntax highlighting. Available options include:
+    <ul>
+      <li>`vim`</li>
+      <li>`monokai`</li>
+      <li>`github`</li>
+      <li>`dracula`</li>
+      <li>And many other standard styles</li>
+    </ul>
+  </dd>
+  
+  <dt>`use_pager`</dt>
+  <dd>Whether to use a pager specifically for syntax-highlighted output (default: `false`)</dd>
+  
+  <dt>`options`</dt>
+  <dd>
+    Additional formatting options:
+    <dl>
+      <dt>`line_numbers`</dt>
+      <dd>Show line numbers in output (default: `false`)</dd>
+      <dt>`wrap`</dt>
+      <dd>Wrap long lines (default: `false`)</dd>
+    </dl>
+  </dd>
+</dl>
+
+### Example Usage
+
+The syntax highlighting is automatically applied when using commands that output YAML or JSON, such as:
+
+```bash
+# Display config in YAML format with syntax highlighting
+atmos describe config -f yaml
+# Display config in JSON format with syntax highlighting
+atmos describe config
+```
+
+<Note>
+When the output is piped to another command, syntax highlighting is automatically disabled to ensure compatibility:
+```bash
+# Syntax highlighting is disabled when piping
+atmos describe config | grep base_path
+```
+</Note>
+
+## Default Settings
+
+If not specified in your `atmos.yaml`, Atmos uses these default settings:
+
+```yaml
+settings:
+  terminal:
+    max_width: 120
+    use_pager: false
+    timestamps: false
+    colors: true
+    unicode: true
+    syntax_highlighting:
+      enabled: true
+      lexer: yaml
```

Comment:
Is lexer still valid? This should be autodetected based on the content, or it's error prone.